FirebaseAuth/Tests/Sample/SwiftApiTests/SwiftAPI.swift

```diff
@@ -87,7 +87,9 @@ class AuthAPI_hOnlyTests: XCTestCase {
     auth.setAPNSToken(Data(), type: AuthAPNSTokenType(rawValue: 2)!)
     auth.canHandleNotification([:])
     try auth.useUserAccessGroup("abc")
-    try auth.getStoredUser(forAccessGroup: "def")
+    let nilUser = try auth.getStoredUser(forAccessGroup: "def")
```

Comment:
We discussed making this an explicit optional type in standup, but after testing it doesn't work because any type `T` can always be cast to `T?`. What this test _should_ express is `nilUser` should not be castable to type `User` without unwrapping.

Reply:
Made this slightly better.